LoRaEngine/LoraKeysManagerFacade/DeviceFeedConstants.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace LoraKeysManagerFacade
+{
+    /// <summary>
+    /// These constants are use when user selected the DEPLOY_DEVICE option while created a Edge Gateway
+    /// This is for demo purpose only and should not be used for productive workloads.
+    /// </summary>
+    internal class DeviceFeedConstants
```

Comment:
This should be a `static` class since it's never expected to be instantiated:

```suggestion
    internal static class DeviceFeedConstants
```

I wonder also why not give it a clear name, like `DemoDeviceIds?` I mean I would make their _demo_ purpose stand out.